### DISCUSSION TITLE
Firebase Realtime Database

### DISCUSSION THREAD

**Discussion by rohitjakhar** [korlibs/korge#1998] [General]
I am just exploring Korge and I just want to know how to connect firebase sdk?
Like i want use firebase featre i.e. Crashlytic, Event, Realtime Database, Remote Config.
Anyone can guide me?

**Comment by ygdrasil-io** [comment]
Korge is a kotlin multiplatform project, so following the doc to KMP on firebase should be the way.

**Comment by soywiz** [comment]
Indeed. There are places where you can search for kotlin libraries. You should search for one supporting the specific SDK you want to use for the specific platforms you want to publish to and include it. Alternatively you can include per-platform libraries for example the Android library, and create expect/actual members yourself to interact with the API.

**Comment by Kietyo** [comment]
You can also use the rest api that it provides and communicate via ktor or korge http apis. Example:

https://gist.github.com/Kietyo/e567ba6dee395b9092ab455362637158
